libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/list/InstructionListPresenter.java

```diff
@@ -157,4 +157,44 @@ private boolean removeInstructionsFrom(int currentInstructionIndex) {
     }
     return false;
   }
+
+  /**
+   * Given the current step / current step distance remaining, this function will
+   * find the current instructions to be shown.
+   *
+   * @param currentStep           holding the current banner instructions
+   * @param stepDistanceRemaining to determine progress along the currentStep
+   * @return the current banner instructions based on the current distance along the step
+   * @since 0.13.0
+   */
+  @Nullable
+  BannerInstructions findCurrentBannerInstructions(LegStep currentStep, double stepDistanceRemaining) {
```

Comment:
```suggestion
  BannerInstructions findCurrentBannerInstructions(@Nullable LegStep currentStep, double stepDistanceRemaining) {
```
will make our life easier in future :)